.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,3 @@
-blank_issues_enabled: false
```

Comment:
can we keep that, our templates are not stable, so having a blank issue can be usefull

Reply:
Done